### ISSUE TITLE
ID 258568: [Programmatic Access - Simon - Table]: Aria attributes are not defined for 'Table’ button under 'Edit Panel' screen.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66335]
### **User Experience:**  
Visually impaired people who depend on screen reader users will not get information about the control as aria attributes are not defined for the control due to which they will not be able to understand the purpose of it and cannot perform any corresponding action.
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:** 

1. Open #URL: [play.grafana.org](https://play.grafana.org/). Sign in with valid credentials. "Home" page appears. 
2. Navigate the Feature Showcase pane heading and select it. 
3. Navigate the expanded Dropdown and select the "Edit" menu item. 
4. Navigate the page "Edit" page which appears. 
5. Navigate to the "All" buttons and select it. 
6. Navigate to "Table" button and expand it. 
7. Run the AI tool. Verify the issue.

**Actual Result:** 
Aria attributes such as `aria-label`, `aria-labelledby` and `title` are not defined for "Table" button under 'Edit Panel' screen.
Screen reader behavior: While navigating on the "Table" button screen reader (Narrator) announces as "Options group table toggle group button expanded".

**Expected Result:** 
Aria attributes such as `aria-label`, `aria-labelledby `and `title` should be defined for "Table" button under 'Edit Panel' screen. Screen reader should announce the `aria-label` as "Table button expanded" when focus moves on it.

_Note: The same issue exists with "Table footer, Standard options and Data links" buttons under same screen. The same issue exists with the controls present under 'All' radio button under Grafana Play Home/ Edit Panel screen. Same issue exists with Value mappings. and thresholds Same issue exists with the table under View rule tab._

**Issue:**
Ensures buttons have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)

**Target application:**  
Grafana Play Home - Grafana - https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&editPanel=7 

**Element path:**  
`button[aria-controls="Table"] Snippet:  <button class="css-vf75o7-button" type="button" aria-expanded="false" aria-controls="Table">`

**How to fix:**
Fix any of the following:   

- Element does not have inner text that is visible to screen readers.   
- `aria-label` attribute does not exist or is empty.   
- `aria-labelledby` attribute does not exist, references elements that do not exist or references elements that are empty.   
- Element has no _title_ attribute.   
- Element's default semantics were not overridden with `role="none"` or `role="presentation"`.

**Environment:**  
Microsoft Edge Version 105.0.1300.0 

This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues.  

### **Severity:**
`SEV2`

### **Area:**
`Panel Edit` (OptionsPaneCategory component)

**Comment by grafanabot** [comment]
@eledobleefe please add one or more appropriate labels. Here are some tips:

- if you are making an issue, TODO, or reminder for yourself or your team, please add one label that best describes the product or feature area. Please also add the issue to your project board. :rocket:

- if you are making an issue for any other reason (docs typo, you found a bug, etc), please add at least one label that best describes the product or feature that you are discussing (e.g. `area/alerting`, `datasource/loki`, `type/docs`, `type/bug`, etc). [Our issue triage](https://github.com/grafana/grafana/blob/main/ISSUE_TRIAGE.md#3-categorizing-an-issue) doc also provides additional guidance on labeling. :rocket:

 Thank you! :heart:
